translated-pairs/en-replace.de.pairs

```diff
@@ -342,48 +342,48 @@ Am Besten
 
 #### Live stats labels shown as the solution runs
 Time:
-Time:
+Zeit:
 
 Solution:
-Solution:
+Lösung:
 
 Run:
-Run:
+Durchlauf:
 
 ### Load/save solution menu
 #### Short names for automatically saved best solutions (try to keep these short)
 ##### A single one can be best in multiple categories
 time, solution, run
-time, solution, run
+Zeit, Lösung, Durchlauf
 
 time, solution
-time, solution
+Zeit, Lösung
 
 time, run
-time, run
+time, Durchlauf
```

Comment:
this "time" is untranslated

Reply:
Damn two mistakes. Sorry 'bout that. Yeah, that's "Zeit, Durchlauf"